usermembership/tool/src/java/org/sakaiproject/umem/tool/ui/UserListBean.java

```diff
@@ -301,6 +300,16 @@ public String getInitValues() {
 		
 		return "";
 	}
+	
+	public TimeZone getUserTimeZone() {
```

Comment:
where is this method being called?

Reply:
In the JSP `timeZone="#{UserListBean.userTimeZone}"`